app/main/autoupdater.js

```diff
@@ -48,6 +50,33 @@ function appUpdater() {
 	autoUpdater.checkForUpdates();
 }
 
+function updateNotification() {
```

Comment:
Can you rename it to something like - `updateDebianNotification` and put it in a different file.

Reply:
What should be the file name?